cpp/src/io/parquet/page_delta_decode.cu

```diff
@@ -742,8 +743,9 @@ CUDF_KERNEL void __launch_bounds__(decode_block_size)
   // For large strings, update the initial string buffer offset to be used during large string
   // column construction. Otherwise, convert string sizes to final offsets.
   if (s->col.is_large_string_col) {
-    compute_initial_large_strings_offset(
-      s, initial_str_offsets[pages[page_idx].chunk_idx], has_repetition);
+    auto const chunks_per_rowgroup = initial_str_offsets.size();
+    auto const input_col_idx       = pages[page_idx].chunk_idx % chunks_per_rowgroup;
```

Comment:
whoops! So, this worked by the grace of the memory pool?

Reply:
Yes, it seems like it.